### PR TITLE
added nginx advanced settings

### PR DESCRIPTION
**Pull request by acarranoqovery**

### CODE REVIEW

File: website/docs/using-qovery/configuration/cluster-advanced-settings.md [hunk 1]
```diff
@@ -83,12 +83,60 @@ Below is the list of advanced settings currently available for clusters.
...
 |---------|-----------------------------------------------------------------------------|---------------|
 | string  | Allows you to specify the load balancer size in front of your cluster.      | `lb-s`        |
 
+### Nginx
+
+#### nginx.vcpu.request_in_milli_cpu ![](/img/advanced_settings/aws.svg) ![](/img/advanced_settings/scaleway.svg) 
+
+| Type    | Description                                                                                    | Default Value   |
+|---------|------------------------------------------------------------------------------------------------|-----------------|
+| integer | Vcpu request value in millicores assigned to Nginx pods                                        | `100`           |
```
Comment (mzottola) [inline website/docs/using-qovery/configuration/cluster-advanced-settings.md:100]:
```suggestion
| integer | Vcpu request value in millicores assigned to Nginx pods                                        | `200`           |
```

File: website/docs/using-qovery/configuration/cluster-advanced-settings.md [hunk 2]
```diff
@@ -83,12 +83,60 @@ Below is the list of advanced settings currently available for clusters.
...
+
+#### nginx.vcpu.limit_in_milli_cpu ![](/img/advanced_settings/aws.svg) ![](/img/advanced_settings/scaleway.svg) 
+
+| Type    | Description                                                                                    | Default Value   |
+|---------|------------------------------------------------------------------------------------------------|-----------------|
+| integer | Vcpu limit value in millicores assigned to Nginx pods                                          | `500`           |
```
Comment (mzottola) [inline website/docs/using-qovery/configuration/cluster-advanced-settings.md:106]:
```suggestion
| integer | Vcpu limit value in millicores assigned to Nginx pods                                          | `700`           |
```

File: website/docs/using-qovery/configuration/cluster-advanced-settings.md (hunk 1 above)
Comment (mzottola) [reply website/docs/using-qovery/configuration/cluster-advanced-settings.md:100]: image

**Comment by acarranoqovery** [comment]
@mzottola , all fixed

**Review by mzottola — APPROVED** [review]

**Pull request merged**